packages/transaction-controller/src/utils.ts

```diff
@@ -1,7 +1,9 @@
+import { Interface } from '@ethersproject/abi';
 import {
   convertHexToDecimal,
   isValidHexAddress,
 } from '@metamask/controller-utils';
+import { abiERC20 } from '@metamask/metamask-eth-abis';
```

Comment:
Is this definitely identical to `human-standard-token-abi'` used in the extension?

Reply:
Yes, both are standard ERC20 abi but just to ensure I diff checked.